JobConfig/cosmic/cosmic_s1_dsstops.fcl

```diff
@@ -229,86 +27,28 @@ physics :
          mcTrajectoryInputs : [ "g4run" ]
          vetoDaughters: []
       }
-
-      g4status:
-      {
-         module_type: FilterStatusG4
-         input: "g4run"
-      }
-
-      g4consistent: {
-         module_type: FilterStatusG4
-         input: "g4run"
-	 maxAcceptedStatus: 9  #  status 10 and above means StepPointMCCollection may have non-dereferencable pointers
-      }
+     g4status: @local::Primary.filters.g4status
```

Comment:
This is outside the scope of this PR, but since we  are using a common g4status configuration it should move to JobConfig/common/mu2eg4.fcl

Reply:
Will do.